### DISCUSSION TITLE
Error while creating the Dev Portal application

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#255] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11.0.2

### Description

Hi!

I am trying to create the Dev Portal application but it fails to create the `dev-portal-micro-lc-docusaurus` repo on Gitlab.
The error I get is the following: "Failed service repository creation: service repository is empty".

Can you help me understanding what is going wrong?
Thanks!

**Comment by peppemanzi** [comment, marked as answer]
Hi, does the account you have configured in your git provider have access to the source repository of the `dev-portal-micro-lc-docusaurus` template?
